### PR TITLE
Feature: Analytics Mobile

### PR DESCRIPTION
**Pull request by Saint-lawal**
### What does this PR do?
- Analytics Mobile.

Screenshot 2020-07-03 at 01 33 41
Screenshot 2020-07-03 at 01 33 48

### CODE REVIEW

File: src/views/public/analytics/Home.vue [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+      <div class="stat w-1/2 lg:w-1/4">
         <p class="stat-title font-semibold font-circular">Parties</p>
         <p class="stat-data">6</p>
         <!-- <p class="stat-data">{{statistics.parties}}</p> -->
       </div>
-      <div class="stat flex-grow pl-10 border-l-2 border-gray-c4">
+      <div class="stat w-1/2 lg:w-1/4 lg:pl-10 lg:border-l-2 border-gray-c4">
         <p class="stat-title font-semibold font-circular">Admins</p>
         <p class="stat-data">6</p>
         <!-- <p class="stat-data">{{statistics.admins}}</p> -->
```
Comment (bolorundurowb) [inline src/views/public/analytics/Home.vue:32]: are we sticking to static data and not live?

Comment (Saint-lawal) [reply src/views/public/analytics/Home.vue:32]: @bolorundurowb I'll leave this for @adetola-ralph

Comment (adetola-ralph) [reply src/views/public/analytics/Home.vue:32]: We're going to use static data where we do not have actual data. I will update as we get live data @bolorundurowb @Saint-lawal

File: src/views/public/analytics/Home.vue [hunk 2]
```diff
@@ -215,10 +215,10 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.statistic-list-container {
-  margin-bottom: 1.25rem;
-  flex: 0 0 calc(50% - 0.625rem);
-}
+// .statistic-list-container {
```
Comment (adetola-ralph) [inline src/views/public/analytics/Home.vue:218]: @Saint-lawal did you check that removing these did not affect the desktop view

**Pull request merged**